darkmod-neo/theme.json

```
@@ -0,0 +1,8 @@
+{
+    "name": "darkmod-neo",
+    "version": "5.0",
+    "description": "DarkMod Neo, A Dark Theme for phpMyAdmin",
+    "author": "Suprethreal",
+    "url": "https://github.com/suprethreal/darkmod-neo/??? ",
```

```suggestion
    "url": "https://github.com/suprethreal",
```
For now because no repository really exits ;)